spec/javascripts/views/story_view_spec.js

```diff
@@ -19,7 +19,7 @@ describe('StoryView', function() {
       notice: sinon.stub(),
       noticeSaveError: sinon.stub()
     };
-    window.md = { makeHtml: sinon.stub() };
+    window.md = { makeHtml: sinon.stub().returns("") };
```

Comment:
When is this being restored? If it's not being restored it'll affect other tests. Somewhere we should return the initial value of `window.md` (not just call `window.md.makeHtml.restore()`, it'll not solve the problem)